cam0:
  T_cam_imu:
  - [0.0002522958025613642, 0.9999880208130867, -0.0048882079695474505, 0.04399491236358173]
  - [-0.99999318319053, 0.00023428420211024648, -0.0036849265099212518, 0.003574658199448571]
  - [-0.003683737137593933, 0.0048891043390562735, 0.9999812631941974, -0.02391335355253343]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [1]
  camera_model: pinhole
  distortion_coeffs: [-0.018667463648839693, -0.007213641011744815, 0.0031810686158049045,
    -0.0017356524706004768]
  distortion_model: equidistant
  intrinsics: [367.0957192620639, 367.19159132903894, 366.0191603533601, 221.21625754751526]
  resolution: [752, 480]
  timeshift_cam_imu: 0.003426703286023108
cam1:
  T_cam_imu:
  - [-0.0013561261629437271, 0.9999152133994791, -0.012950943367376776, -0.07587403543051402]
  - [-0.999995536827879, -0.001321529710947844, 0.0026795304711607366, 0.003874473915850829]
  - [0.002662188226436303, 0.012954519346463739, 0.9999125427667912, -0.024023615450020193]
  - [0.0, 0.0, 0.0, 1.0]
  T_cn_cnm1:
  - [0.9999661999878838, 0.0016381045309823115, -0.008057015287236475, -0.12006598668445986]
  - [-0.001586906658822055, 0.999978536562433, 0.006356739865115712, 0.0005217192280754045]
  - [0.008067255360367197, -0.0063437392760217, 0.9999473367947684, -0.00044376274430848664]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [-0.020781215249804308, -0.0010709270389364762, -0.00285240050393458,
    0.00020259489968586973]
  distortion_model: equidistant
  intrinsics: [366.3718254073452, 366.46119617211144, 387.9728603362247, 250.71395852424962]
  resolution: [752, 480]
  timeshift_cam_imu: 0.003185970669502835
T_imu_body:
  [1.0000, 0.0000, 0.0000, 0.0000,
  0.0000, 1.0000, 0.0000, 0.0000,
  0.0000, 0.0000, 1.0000, 0.0000,
  0.0000, 0.0000, 0.0000, 1.0000]